src/plugins/twitter.py

```diff
@@ -69,7 +76,13 @@ def send_payout_notification(self, cycle, payout_amount, nb_delegators):
         # Truncate message to max tweet length
         tweet = tweet[: self.MAX_TWEET_LEN]
 
-        resp = self.twitter.update_status(tweet)
+        # We use APIv2 to post, API 1.1 no longer allow posting tweet
+        client = tweepy.Client(
```

Comment:
line 32 says "self.twitter = tweepy.API" what is this self.twitter used for?

Reply:
it initialize tweepy API, self-twitter used on line bellow to get the displayname of the twitter account